mlflow/cli.py

```diff
@@ -281,6 +282,34 @@ def server(backend_store_uri, default_artifact_root, host, port,
         sys.exit(1)
 
 
+@cli.command()
+@click.option("--backend-store-uri", metavar="PATH",
+              default=DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH,
+              help="URI to which to persist experiment and run data. Acceptable URIs are "
+                   "SQLAlchemy-compatible database connection strings "
+                   "(e.g. 'sqlite:///path/to/file.db') or local filesystem URIs "
+                   "(e.g. 'file:///absolute/path/to/directory'). By default, data will be deleted "
+                   "from the ./mlruns directory.")
+@click.option("--run-ids", default=None,
+              help="Comma separated list of runs to be permanently deleted.")
+def gc(backend_store_uri, run_ids):
```

Comment:
Can we also add the `experimental` decorator to this method?

Reply:
Done